src/main/java/io/github/nucleuspowered/nucleus/modules/ban/commands/TempBanCommand.java

```diff
@@ -34,10 +40,27 @@
 @NoCost
 public class TempBanCommand extends CommandBase<CommandSource> {
 
+    @Inject private BanConfigAdapter bca;
     private final String user = "user";
     private final String reason = "reason";
     private final String duration = "duration";
 
+    private final String bypassMaximumLength = PermissionRegistry.PERMISSIONS_PREFIX + "tempban.bypass";
```

Comment:
Is this an exemption? The permission node is used to bypass the maximum time when temporarily banning another user.